src/app/components/client/exposure_card/ScanResultCard.tsx

```diff
@@ -246,15 +246,12 @@ export const ScanResultCard = (props: ScanResultCardProps) => {
             className={styles.chevron}
             onClick={() => props.onToggleExpanded()}
             aria-expanded={props.isExpanded}
+            aria-label={`${l10n.getString("chevron-alt")} ${props.isExpanded ? l10n.getString("chevron-expanded") : l10n.getString("chevron-collapsed")}`}
```

Comment:
As I understood the ticket, we don't need to manually add "Expanded" or "Collapsed" to this string - the screen reader will announce that by itself, thanks to the `aria-expanded` attribute. So currently when I use my screen reader, this announces "Exposure details collapsed collapsed".

(I'd also expect this label to be used as the `alt` for the image as we did before, as `aria-label` is a last resort for when there is no content, but I think in practice the effect will be the same.)

```suggestion
            aria-label={l10n.getString("chevron-alt")}
```

Reply:
You can just select the button with the same string. If you want to verify whether the button indicates expanded/collapsed state, you could e.g. do

```
expect(expandButton).toHaveAttribute("aria-expanded", "true")
```